ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml

```diff
@@ -557,10 +579,30 @@ periodics:
       - name: CLUSTER_TYPE
         value: aws
       - name: CONFIG_SPEC
-        valueFrom:
-          configMapKeyRef:
-            key: openshift-origin-release-4.4.yaml
-            name: ci-operator-4.4-configs
+        value: |
+          tag_specification:
+            name: "none" # don't import anything to avoid a race between 4.2 latest and the chosen version
```

Comment:
You shouldn't race anything.  I think this is structured wrong.  You shouldn't even have to create a release image.  What you need to do is run ci-operator twice - once before you do the builds to do the imports, then run your build to override the image, then run it again and it will generate the payload for you.

So your definition should look like a standard okd branch definition, minus the build jobs.  You want to run to `--target=[images]`, then you run your build and have it override `stable:machine-os-content`.

Reply:
So I tried that - and I can't make the script find the create namespace. It appears `namespace="ci-op-$( echo "${JOB_NAME}-${BUILD_ID}" | md5sum -t | cut -c 1-8 )"` doesn't work, cause `ci-tools` now includes a timestamp.